templates.dist/fields.php

```diff
@@ -269,24 +277,37 @@ function renderParkingLot(array $parkingArray, string $language, string $locale,
 if (!function_exists('formatPriceParking')) {
 	function formatPriceParking(string $str, string $language, string $locale, string $codeCurrency, string $currency): string
 	{
+		$listCurrency = [
+			'EUR' => 'de_DE',
+			'CZK' => 'cs_CZ',
+			'TRY' => 'tr_TR',
+			'BGN' => 'bg_BG',
+			'PLN' => 'pl_PL',
+			'RON' => 'ro_RO',
+			'SEK' => 'sv_SE',
+			'HRK' => 'hr',
+			'GBP' => 'en_GB',
+			'CHF' => 'fr_FR',
+			'USD' => 'en_US',
+			'HUF' => 'hu_HU',
+		];
 		$digit = intval(substr(strrchr($str, "."), 1));
 		if (class_exists(NumberFormatter::class)) {
 			$format = new NumberFormatter($locale, NumberFormatter::CURRENCY);
-			if ($digit) {
-				$format->setAttribute(NumberFormatter::MAX_FRACTION_DIGITS, 2);
+			if ( $digit ) {
+				$format->setAttribute( NumberFormatter::MAX_FRACTION_DIGITS, 2 );
 			} else {
-				$format->setAttribute(NumberFormatter::MAX_FRACTION_DIGITS, 0);
-			}
-			if (empty($codeCurrency))
-			{
-				$codeCurrency = "EUR";
+				$format->setAttribute( NumberFormatter::MAX_FRACTION_DIGITS, 0 );
 			}
+			if ( $currency == '?' ) {
```

Comment:
I don't understand why there is so much code around the currency. We should simply get the estate's currency and use that.